### PR TITLE
Submitting NavigraphCharts  version 8.25.2

### PR DESCRIPTION
**Pull request by kmc00**
- [/Y] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ Y] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ Y] This PR only modifies one (1) manifest
- [ Y] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ Y] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ Y] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106566)

### CODE REVIEW

**Comment by kmc00** [comment]
@microsoft-github-policy-service agree

**Comment by kmc00** [comment]
think the prompt saying "Installed" at the end on the install, might be causing the pipe line to fail.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x64-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230512011231

**Comment by stephengillie** [comment]
Manual Validation failed with:
> 11:44:26 AM: 2023-05-12 11:43:45.999 [CLI ] Package hash verification failed. SHA256 in manifest 
> [21264754cbe8a5f9135bc16c24d275e1bfc1cbb7d476a9c60cf9e8bbb37109cf] does not match download 
> [c73deb09801ee2e086948f42aeba3502e0b3603347a65716520e9a3d53b04257]

File: manifests/n/Navigraph/NavigraphCharts/8.25.2/Navigraph.NavigraphCharts.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Navigraph.NavigraphCharts
+PackageVersion: 8.25.2
+Installers:
+- InstallerUrl: https://download.navigraph.com/software/charts/win/Navigraph%20Charts.exe
+  Architecture: neutral
+  InstallerType: nullsoft
+  InstallerSha256: 21264754CBE8A5F9135BC16C24D275E1BFC1CBB7D476A9C60CF9E8BBB37109CF
```
Comment (stephengillie) [inline manifests/n/Navigraph/NavigraphCharts/8.25.2/Navigraph.NavigraphCharts.installer.yaml:10]:
```suggestion
  InstallerSha256: C73DEB09801EE2E086948F42AEBA3502E0B3603347A65716520E9A3D53B04257
```

**Comment by stephengillie** [comment]
This package almost looks like a portable application from its silent installer. But the installer launches the Navigraph Simlink installer, which is not unattended. Were this package preinstalled, it's possible that this package would install silently.

**Comment by kmc00** [comment]
Don't worry about it for now.
Will come up with a fix

**Comment by kmc00** [comment]
Thanks for the help @stephengillie

**Pull request closed without merging**